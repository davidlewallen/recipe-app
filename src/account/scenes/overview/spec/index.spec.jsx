import React from 'react';
import { shallow } from 'enzyme';

import Overview from '../components';

describe('Overview', () => {
  it('should render and match snapshot', () => {
    const wrapper = shallow(
      <Overview user={{ username: 'testName', email: 'test' }} />
    );
    expect(wrapper.find('Grid').exists()).toBe(true);
    expect(wrapper).toMatchSnapshot();
  });
});
