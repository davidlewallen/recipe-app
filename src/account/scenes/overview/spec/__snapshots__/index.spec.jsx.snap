// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overview should render and match snapshot 1`] = `
<Grid
  bsClass="container"
  className="overview"
  componentClass="div"
  fluid={false}
>
  <Row
    bsClass="row"
    componentClass="div"
  >
    <Col
      bsClass="col"
      componentClass="div"
      md={4}
      mdOffset={4}
      xs={12}
    >
      <form>
        <FormGroup
          bsClass="form-group"
          controlId="formAccountInfo"
        >
          <ControlLabel
            bsClass="control-label"
            srOnly={false}
          >
            Username
          </ControlLabel>
          <FormControl
            bsClass="form-control"
            componentClass="input"
            disabled={true}
            type="text"
            value="testName"
          />
          <ControlLabel
            bsClass="control-label"
            srOnly={false}
          >
            Email
          </ControlLabel>
          <FormControl
            bsClass="form-control"
            componentClass="input"
            disabled={true}
            type="text"
            value="test"
          />
        </FormGroup>
      </form>
    </Col>
  </Row>
</Grid>
`;
